scraper.ts

```diff
@@ -0,0 +1,128 @@
+// Reads features-json/*.json files from caniuse repository
+// and writes each file in a "respec friendly way"
+//  - Keep only the `stats` from features-json data
+//  - Sort browser versions (latest first)
+//  - Remove footnotes and other unnecessary data
+
+import { promises as fs, existsSync } from 'fs';
+import { spawn } from 'child_process';
+import path from 'path';
+import { DATA_DIR } from './constants.js';
+
+const { readFile, writeFile, readdir, mkdir } = fs;
+
+interface Input {
+  stats: {
+    [browserName: string]: { [version: string]: string };
+  };
+}
+
+interface Output {
+  [browserName: string]: [string, ReturnType<typeof formatStatus>][];
+}
+
+const INPUT_DIR = path.join(DATA_DIR, './caniuse-raw/features-json/');
+const OUTPUT_DIR = path.join(DATA_DIR, './caniuse/');
+
+export async function main() {
```

Comment:
this is updated as "no longer a process". This can be imported in respec.org (maybe in a Worker)